sdv/constraints/base.py

```diff
@@ -226,6 +227,14 @@ def _validate_constraint_columns(self, table_data):
         """
         missing_columns = [col for col in self.constraint_columns if col not in table_data.columns]
         if missing_columns:
+            if not self._columns_model:
+                warning_message = (
```

Comment:
Could we add a small unit test for this?

Reply:
Yes, I just added it!

Reply:
Awesome. Now that I look at this, I wonder if it is confusing to a user. @csala The case where this happens is usually because of conditioning right? Would they be confused to see a warning that says "constraints are missing from table data". Should we say that it's because of conditioning to provide more context?

Reply:
Hm that's a good point. I didn't completely understand the use case when I wrote this message.

I can update the comment to say, "When `fit_columns_model` is `False` and we are conditioning on a subset of the constraint columns, reject sampling can become slow."

What do you think?

Reply:
I think that makes sense